cfn-resources/project/cmd/resource/resource.go

```diff
@@ -30,11 +31,8 @@ import (
 	"go.mongodb.org/atlas/mongodbatlas"
 )
 
-var CreateRequiredFields = []string{constants.PubKey, constants.PvtKey, constants.OrgID, constants.Name}
-var ReadRequiredFields = []string{constants.PubKey, constants.PvtKey}
-var UpdateRequiredFields = []string{constants.PubKey, constants.PvtKey, constants.ID}
-var DeleteRequiredFields = []string{constants.PubKey, constants.PvtKey}
-var ListRequiredFields = []string{constants.PubKey, constants.PvtKey}
+var CreateRequiredFields = []string{constants.OrgID, constants.Name}
+var UpdateRequiredFields = []string{constants.ID}
```

Comment:
[q] Wouldn't `Profile` be a required property for every resource operation, otherwise how will they know which profile to use?

Reply:
Answered on standup - primary identifier does not need to be set as required.